### ISSUE TITLE
[Bug]: Crash on starting up the game

### ISSUE THREAD

**Issue by MacatMovie** [Toadstool-Studios/Creeper-Overhaul#111]
### Bug Description

Been updating a huge modpack to 1.20.1 from 1.19.2 and it seems like CoFH Thermal Mods won't work with Creeper Overhaul mod. Tried on previously used and new worlds. Here's the crash report:
https://pastebin.com/gjRAHB8Q

### How to Reproduce?

1. Open Game
2. Crash when loaded.

### Version

3.0.1

### Mod Loader Version

47.2.0

### Mod Loader

Forge

### Logs or additional context

[crash-2023-12-25_18.22.56-client.txt](https://github.com/bonsaistudi0s/Creeper-Overhaul/files/13766883/crash-2023-12-25_18.22.56-client.txt)

### Acknowledgement

- [x] I agree that I have read what was said above this check box.

**Comment by ModernGamingWorld** [comment]
I just ran into this myself

**Comment by windth001** [comment]
I went to the CoFH Team's Discord. People are reporting a conflict with Creeper Overhaul there as well. 
It was suggested to: Disable Creeper Overhaul's option to override the default creeper model or Disable the Thermal's vanilla holiday hats in the 'thermal-client' configs.

According to the persons who suggested this as a "fix" Creeper Overhaul has a client side-only functionality that is conflicting with the Thermal series creating an incompatibility.

"Creeper Overhaul replaces the creepers model with one that doesn't say its alive, and Thermal adds a hat to some mobs that say they are alive, but with the creepers model replaced the creeper says its alive but the model for the creeper says it isn't, so thermal gets confused"

Confirmed. Setting the holiday hats in the 'thermal-client' configs to false "fixed" the issue.
